test/RestAPI/Topics/library.test.js

```diff
@@ -44,21 +54,167 @@ describe('Testing Library module', () => {
     });
 
     it('Automatically includes auth-token, if module already has a token stored', () => {
-      const fakeAuthToken = 'someAuthToken';
+      const requestOptions = expect.objectContaining({
+        headers: expect.objectContaining({
+          'X-MSTR-AuthToken': sharedInputInfo.fakeAuthToken
+        })
+      });
+      expect(axios).toHaveBeenCalledWith(requestOptions);
+    });
+  });
+
+  describe('publishObject()', () => {
 
-      // either call login API via module to store token, or manually provide a token to store
-      const mstrApi = new mstr.REST(sharedRestOptions);
-      mstrApi.setAuthToken(fakeAuthToken);
+    let requestAction;
```

Comment:
These requestAction variables are only used within the beforeEach callback. It doesn't seem necessary to store them outside the beforeEach scope, or am I missing something?

Reply:
I read this as a solution. This way you can reference in your tests, otherwise it is scoped to the beforeEach block.

Reply:
Does it break if you put the var into the beforeEach block? Can you add a comment with a short explanation on why that var must stay outside?

Reply:
Removed the var after our discussion in Slack

Reply:
vars are still there, did you commit push?